docs/getting-started/index.md

```diff
@@ -7,8 +7,30 @@ slug: /getting-started
 
 # Getting Started
 
+## Overview
+
 Learn how to set up a Linux system to use zoned storage devices.
 
+## Introduction
+
+If you have never used a Shingled Magnetic Recording (SMR) disk before, this is
+the chapter that you should read. 
+
+## null_blk Emulation Setup
+
+The following three-step procedure results in an environment that will allow
+you to evaluate ZonedStorage and to investigate its features. This is the
+fastest way to get started, and is recommended for first-time readers.
+
+This procedure creates a zoned null block device. It is okay if you do not yet
+know what that means. This is the simplest way to try out ZonedStorage.
+
+1. Install a Linux distribution that supports ZonedStorage. Fedora 36 is recommended.
+1. Install ``lsblk`` and ``blkzone`` on the Linux distribution that supports ZonedStorage. In Fedora 36, the package called ``util-linux`` provides ``lsblk`` and ``blkzone``.
```

Comment:
There is no need to install lsblk and blkzone. These are part of util-linux sys-utils which is installed by default, always, on any Linux distro.

Reply:
Roger that. I've removed it.

Reply:
The text you have is OK and simple, but I do not like very much how the getting started button landing page now looks. It is messy... We need to think how to better integrate this, or redesign the getting started landing page.
Let me try a few things (I have pulled locally your patch, through git cli).

Reply:
My plan was to get these tutorial sections into the documentation and then streamline the landing page.